docs/application/native/tutorials/details/native-removed-history.md

```diff
@@ -0,0 +1,39 @@
+# Removed functions and enumerations of Tizen Native API
+
+This page describes the functions and enumerations that were removed in Tizen 4.0 and 5.0.
+
+If you want to know the lifecycle of Tizen Native API, you need to read the contents of [API Versioning and Deprecation Policy of the Tizen Platform](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/api-versioning-and-deprecation-policy-tizen-platform) first.
+
+## Detailed information regarding removed functions and enumerations
+
+| Module | Items(s) | Profile | Deprecated | Removed | Reason | 
+| ------- | ------- | ------ | ---- | ---------- | ----------- |
+| Security - OpenSSL | SSLv2_method(), SSLv2_client_method(), SSLv2_server_method() | Mobile & Wearable | - | 3.0 | Security |
+| Base - Utils - i18n | i18n_timezone_in_daylight_time(), i18n_usearch_create(), i18n_ustring_to_title() | Mobile & Wearable | since 2.3.1 | 4.0 | No longer available | 
+| Web - WebView | ewk_context_preferred_languages_set(), ewk_settings_private_browsing_enabled_set(), ewk_settings_private_browsing_enabled_get() | Mobile | since 2.4 | 4.0 | No longer available |
+| Base - Glib | Gdbus APIs (g_bus_\*(), g_dbus_\*(), and g_test_dbus_\*())| Mobile & Wearable | since 2.3.2 | 5.0 | Security |
+| Content - Media Content | audio_meta_get_played_count(), audio_meta_get_played_time(), audio_meta_get_played_position(), audio_meta_set_played_count(), audio_meta_set_played_time(), audio_meta_set_played_position(), media_content_set_db_updated_cb(), video_meta_get_played_time(), video_meta_get_played_position(), video_meta_set_played_count(), video_meta_set_played_time(), video_meta_set_played_position()|Mobile & Wearable | since  2.4 | 5.0 | Better alternatives | 

```

Comment:
`Better alternatives` - How about writing to which alternative it is changed?

Reply:
It is diffcult to describe all alternatives in this document.
I will consider that....